Code/skyrim_ui/src/app/components/settings/settings.component.ts

```diff
@@ -78,12 +74,6 @@ export class SettingsComponent implements OnInit {
     this.language = this.settingService.getLanguage();
   }
 
-  onShowDebugChange(checked: boolean) {
-    this.settingService.setDebugShown(checked);
-    this.client.debugStateChange.next(checked);
```

Comment:
There are no listeners for ClientService.debugStateChange, so I don't see why it can't be removed. It seems to be overwriting a variable sent by the client, which is never used and changing it in the UI code will not send the changes back to the Client.

Reply:
Cool, that's fine then, figured it was worth checking.